wg-releases/sudowoodo.md

```diff
@@ -0,0 +1,30 @@
+## Releasing Electron
+
+Electron's Release process is performed using a Slack-based bot system called Sudowoodo, which lives [here](https://github.com/electron/sudowoodo).
+
+### Running Sudowoodo
+
+Only authorized releasers have the ability to trigger Sudowoodo, and they can be seen and verified by running `/sudowoodo-trainers` from within Electron's Slack workspace.
+
+In order to trigger a new release:
+
+1. Invoke `/sudowoodo` as a backslack command in the `#bot-releases` Slack channel.
+2. Choose the correct release branch (`master`, `5-0-x`, etc.) and release type (`stable`, `beta`, `nightly`) from the dropdown in the resulting dialog.
+3. Monitor the resulting release until builds are finished and Sudowoodo is preparing to publish the new release to npm
+4. Enter the OTP needed to publish the builds to npm (similarly gated to `sudowoodo-trainers`)
+5. Celebrate 🎉
+
+### Troubleshooting Sudowoodo
+
+Sometimes Sudowoodo hits roadblocks and can't complete the release on its own. In nearly all cases, these roadblocks fall into one of a small number of categories.
+
+1. CircleCI timeouts
+  * This happens sometimes as a result of failures of CircleCI infrastructure that cause builds to be rerun within their containers. 
+  * Sudowoodo has built-in resilience to this.
+  * Should this occur, the release process can be continued by invoking `/blasting-off-again {COMMIT_HASH}` as a backslash command in `#bot-releases`.
```

Comment:
We should mention here that you should only run `/blasting-off-again` once all the builds that failed have been rerun and completed successfully.  Maybe we should also explain what to use as the `COMMIT_HASH` here